src/plugins/gui/pluginUtils.ts

```diff
@@ -48,32 +39,53 @@ export const getBestError = (errorQuotes: FiatProviderError[], currencyCode: str
       bestError = errorQuote
       continue
     }
-    // @ts-expect-error
     if (ERROR_PRIORITIES[errorQuote.errorType] < ERROR_PRIORITIES[bestError.errorType]) {
       bestError = errorQuote
       continue
     }
-    // @ts-expect-error
     if (ERROR_PRIORITIES[errorQuote.errorType] === ERROR_PRIORITIES[bestError.errorType]) {
       if (errorQuote.errorType === 'overLimit' && bestError.errorType === 'overLimit') {
-        if (errorQuote.errorAmount > bestError.errorAmount) {
+        if (bestError.errorAmount != null && errorQuote.errorAmount != null && errorQuote.errorAmount > bestError.errorAmount) {
```

Comment:
Should also handle the case where `bestError.errorAmount == null && errorQuote.errorAmount != null'

Change the `if` to

```
if ((errorQuote.errorAmount ?? 0) > (bestError.errorAmount ?? 0)) {
```